docs/guidance/gcs-ft.md

```diff
@@ -67,6 +67,16 @@ Also, you can specify a storage namespace for your Ray cluster by using an annot
 
 An example can be found at [ray-cluster.external-redis.yaml](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-cluster.external-redis.yaml)
 
+To use SSL/TLS in the connection, the following environment variables are needed:
+
+- `RAY_REDIS_CA_CERT` The location of the CA certificate (optional)
+- `RAY_REDIS_CA_PATH` Path of trusted certificates (optional)
+- `RAY_REDIS_CLIENT_CERT` File name of client certificate file (optional)
+- `RAY_REDIS_CLIENT_KEY` File name of client private key (optional)
+- `RAY_REDIS_SERVER_NAME` Server name to request (SNI) (optional)
+
+Besides, `rediss://` should be added as the prefix of the redis address. Ray 2.2 is needed for this feature.
```

Comment:
```suggestion
To use SSL/TLS in the connection, you add `rediss://` as the prefix of the redis address instead of the `redis://` prefix. This feature is only available in Ray 2.2 and above.

You can also specify additional environment variables in the head pod to customize the SSL configuration:

- `RAY_REDIS_CA_CERT` The location of the CA certificate (optional)
- `RAY_REDIS_CA_PATH` Path of trusted certificates (optional)
- `RAY_REDIS_CLIENT_CERT` File name of client certificate file (optional)
- `RAY_REDIS_CLIENT_KEY` File name of client private key (optional)
- `RAY_REDIS_SERVER_NAME` Server name to request (SNI) (optional)

```